packages/genesys-spark-components/src/components/stable/gux-dismiss-button/gux-dismiss-button.tsx

```diff
@@ -30,7 +36,10 @@ export class GuxDismissButton {
     return (
       <gux-button-slot
         accent="ghost"
-        class={this.position == 'inherit' ? 'gux-inherit' : undefined}
+        class={{
+          'gux-inherit': this.position == 'inherit',
```

Comment:
should we have a `===` instead?

Reply:
I'd prefer to change as little code as possible so I'll keep it that way, but I do think in this case it's unlikely to be much of a difference, especially with the variable having specific strings for types.